corehq/motech/repeaters/views/tests/test_repeat_records.py

```diff
@@ -115,6 +115,7 @@ class view:
             # updated if the keys in this dict change
             'Pending': 1,
             'Fail': 0,
+            'InvalidPayload': 0,
```

Comment:
re: comment above. Do template(s) that reference `count_State` need to be updated?

https://github.com/dimagi/commcare-hq/blob/acf5951647819fe41ce6b1e514ab5137dd9a663d/corehq/motech/repeaters/templates/repeaters/partials/repeater_row.html#L11-L26

Reply:
At first I added a new column for invalid payloads, but the table didn't look good for repeaters with long names. Considering we already exclude the Empty state, I thought it would be better to exclude the Invalid Payload state too.

But on second thoughts that seems wrong. So I've included the Invalid Payload state with Cancelled (and Empty with Success). https://github.com/dimagi/commcare-hq/pull/34999/commits/d6924defebd4313389d5e21c401a1d0526a3f52b

There is a downside though. The link for the count of "InvalidOrCancelled" opens the repeat records report with the filter set to Cancelled. It's not going to be obvious to users that they need to change the filter to "Invalid Payloads" to see the other repeat records. What do you think? Another column? Or Success-ish and Cancelled-ish?

Reply:
I don't have much to add here, but I think this table is already notoriously confusing for users. I don't mind decoupling the states that a user sees from the states that exist in the backend, which I think would be success-ish and cancelled-ish in this case.